spk/jackett/Makefile

```diff
@@ -1,33 +1,38 @@
 SPK_NAME = jackett
-SPK_VERS = 0.7.1483
-SPK_REV = 4
+SPK_VERS = 0.8.490
+SPK_REV = 5
 SPK_ICON = src/jackett.png
 DSM_UI_DIR = app
 
 DEPENDS = cross/busybox cross/curl cross/$(SPK_NAME)
-SPK_DEPENDS = "mono>=4.2.1.102-6"
+# depend on mono 5.4 for BTLS HttpWebClient support
+SPK_DEPENDS = "mono>=5.4"
 
-MAINTAINER = SynoCommunity
-DESCRIPTION = "Jackett works as a proxy server: it translates queries from apps into tracker-site-specific http queries, parses the html response, then sends results back to the requesting software. Jackett is a single repository of maintained indexer scraping & translation logic - removing the burden from other apps."
-ADMIN_PORT = 9117
+MAINTAINER = 'kaso17 (Jackett), m4tt075 (spksrc integration)'
```

Comment:
I wonder if `spkrepo` will accept packages with such maintainer. As far as I understood/remember, it has to match a user account registered in registry